README.rdoc

```diff
@@ -55,11 +55,11 @@ Also, if you pass the -r option, it'll annotate routes.rb with the output of
 
 Into Gemfile from rubygems.org:
 
-    gem 'annotate'
+    gem 'annotate', require: false
 
 Into Gemfile from Github:
 
-    gem 'annotate', git: 'https://github.com/ctran/annotate_models.git'
+    gem 'annotate', github: 'ctran/annotate_models', require: false
```

Comment:
This block is which was declared on the top of Gemfile:

```ruby
git_source(:github) {|repo_name| "https://github.com/#{repo_name}.git" }
```

Its a workaround for a bug in Bundler which can cause sources from github to loaded via HTTP and not HTTPS - which makes it vulnerable to man in the middle attacks. We'll usually not remove it.